docs/send-data/opentelemetry-collector/auto-discovery.md

```diff
@@ -0,0 +1,47 @@
+---
+id: auto-discover
+title: Auto Discovery
+sidebar_label: Auto Discovery
+description: Learn how to enable Auto Discovery to detect services that are installed on the server on which the collector is running.
+---
+
+Sumo Logic OpenTelemetry collectors can now detect services that are installed on the server on which the collector is running. Once the services are discovered, you can evaluate the information on Sumo Logic platform and proceed with the onboarding of applications. It's a useful improvement that will give you more confidence in monitoring and observability of your systems.
```

Comment:
This intro reads more like a release note than a doc. I'd def use this for the release note. I'd eliminate "_can now_" and "_useful improvement_" so that the content is evergreen. So something like...

```suggestion
With the Sumo Logic OpenTelemetry collector Auto Discovery feature, you can detect, monitor, and observe services that are installed on the server on which the collector is running. Once the services are discovered, you can evaluate the information on Sumo Logic platform and proceed with the onboarding of applications.
```